### DISCUSSION TITLE
Add Heilbronn Campus labs and lecture halls

### DISCUSSION THREAD

**Discussion by mohanadkandil** [TUM-Dev/NavigaTUM#3052] [Data]
**Is your feature request related to a problem? Please describe.**
There's no info about offices/lecture halls/labs etc in our campus.

**Describe the solution you'd like**
Gather all data with images and locations and add them to NavigaTUM.

**Comment by CommanderStorm** [comment]
Currently we get our data from patches and other roomfinders.
=> see https://nav.tum.de/about/about-us for a list of our sources

Is there a Roomfinder/other publicly scrapable source for the Heilbronn campus we can access?
(As I am not located there and TUM does not cover travel-expenses for my position, going there and looking is not feasible)

I think for the scale of the Heilbronn campus (and the low amount of different lecture halls):
Is a roofinder even helpfull?
